store/sqlstore/supplier.go

```diff
@@ -911,7 +911,7 @@ func (ss *SqlSupplier) createIndexIfNotExists(indexName string, tableName string
 
 		_, err = ss.GetMaster().ExecNoTimeout("CREATE  " + uniqueStr + fullTextIndex + " INDEX " + indexName + " ON " + tableName + " (" + strings.Join(columnNames, ", ") + ")")
 		if err != nil {
-			mlog.Critical("Failed to create index", mlog.Err(err))
+			mlog.Critical("Failed to create index", mlog.String("table", tableName), mlog.Err(err))
```

Comment:
Should we include the index name while we're touching this?

Reply:
I was going to include it but didn't if it was interesting. Given that looks interesting to include it I'll do :)